xc/stubdom.ml

```diff
@@ -77,9 +77,10 @@ let build (task: Xenops_task.task_handle) ~xc ~xs ~dm ~store_domid ~console_domi
   let path = Printf.sprintf "/vm/%s/image/dmargs" guest_uuid_str in
   (* Remove any 'pty' references from the arguments: XXX why? *)
   let info' = { info with Device.Dm.serial = None; monitor = None } in
-  let args = Device.Dm.cmdline_of_info ~xs ~dm info' false domid in
-  xs.Xs.write path (String.concat " " args);
```

Comment:
@rossla do you know why qemu args are added to xenstore?

Reply:
This could be wrong and a result of re-basing my code.